boards/shields/x_nucleo_iks01a3/x_nucleo_iks01a3.overlay

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2019 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&arduino_i2c {
+
+	hts221@5f {
+		compatible = "st,hts221";
+		reg = <0x5f>;
+		label = "HTS221";
+	};
+
+	lps22hh@5d {
+		compatible = "st,lps22hh";
+		reg = <0x5d>;
+		drdy-gpios = <&gpiob 10 0>;
```

Comment:
Can you look at the `arduino_header` introduced in #14057 and commit ce1e990c30f82be8a5c61a41dac1addf317c9538 , and make these gpios not specific to stm32 socs?

Reply:
You mean the drdy gpios? Yes, excellent idea!
I will define arduino_header for nucleo F401re and F411re for testing purpose, and I will
change this shield overlay file to make use of it.